google/cloud/bigtable/table_admin.cc

```diff
@@ -95,6 +96,54 @@ void TableAdmin::DropAllRows(std::string const& table_id) {
   }
 }
 
+std::future<btadmin::Snapshot> TableAdmin::SnapshotTable(
+    bigtable::ClusterId const& cluster_id,
+    bigtable::SnapshotId const& snapshot_id, bigtable::TableId const& table_id,
+    ::google::protobuf::Duration duration_ttl) {
+  return std::async(std::launch::async, &TableAdmin::SnapshotTableImpl, this,
+                    cluster_id, snapshot_id, table_id, duration_ttl);
+}
+
+btadmin::Snapshot TableAdmin::SnapshotTableImpl(
+    bigtable::ClusterId const& cluster_id,
+    bigtable::SnapshotId const& snapshot_id, bigtable::TableId const& table_id,
+    ::google::protobuf::Duration duration_ttl) {
+  // Copy the policies in effect for the operation.
+  auto rpc_policy = impl_.rpc_retry_policy_->clone();
```

Comment:
These are not used?  Why clone them here?

Reply:
removed.